substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1078,6 +1079,13 @@ void handleModuleAttributes(Attributes mainAttributes) {
         }
     }
 
+    void handleEnableNativeAccessAttribute(Attributes mainAttributes) {
+        String enableNativeAccess = mainAttributes.getValue("Enable-Native-Access");
+        if (enableNativeAccess != null) {
```

Comment:
We should check here that `enableNativeAccess` is `ALL-UNNAMED` because that is the only eligible values and if it is not, issue a error message similar to the JDK.

Reply:
Done